test/testsuite/e2e_suite.go

```diff
@@ -15,22 +15,24 @@ func (s *E2ETestSuite) SetupXCall(ctx context.Context, portId string) error {
 		return err
 	}
 	var err error
-	_, err = chainA.ConfigureBaseConnection(context.Background(), chains.XCallConnection{
+	_, err = chainA.ConfigureBaseConnection(ctx, chains.XCallConnection{
 		KeyName:            Owner,
 		CounterpartyNid:    chainB.(ibc.Chain).Config().ChainID,
 		ConnectionId:       "connection-0", //TODO
```

Comment:
todo detected on hard coded value here. is it possible to pass it via args?

Reply:
As we intend to establish a connection only once for each xCall, I guess it seems reasonable to use a hardcoded connection ID.